plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/data/JDBCContentLOB.java

```diff
@@ -109,4 +114,15 @@ public DBDValueCloneable cloneValue(DBRProgressMonitor monitor)
 
     protected abstract JDBCContentLOB createNewContent();
 
+    void handleContentReadingException(DBCException e) throws DBCException {
+        DBCTransactionManager transactionManager = DBUtils.getTransactionManager(executionContext);
+        if (!errorMessageIsShown && transactionManager != null && transactionManager.isAutoCommit()) {
+            DBWorkbench.getPlatformUI().showWarningMessageBox(
+                ModelMessages.jdbc_content_view_error_message_title,
+                ModelMessages.jdbc_content_view_error_message_body);
+            errorMessageIsShown = true;

```

Comment:
I don't like that model communicates with UI. This doesn't seem right.

It would be best if you handled exceptions in the UI. See the `ContentPanelEditor` class.